content/release-notes/2022-12-01.md

```diff
@@ -0,0 +1,38 @@
+---
+label: 'Console'
+---
+
+### What's new
+
+- Control plane: List operations in V2 API supports cursor pagination.
+- Control plane: Added validation for empty names for branches and 
+projects. 
+- Control plane: Added LSN validation.
+- Console: Added 'delete endpoint' button.
+- Control plane: Make parent_id and name optional when creating a branch.
+- Console: Show branch_id and fork button in branch view.
+- Control plane: Added new autoscaling limits.
+- Control plane: Added region maintenance/testing flag.
+- Control plane: Return complete host for connecting in endpoint response.
+- Control plane: Rename default database from `main` to `neondb`.
+- Control plane: Add per-project branch/endpoint limits.
```

Comment:
```suggestion
- UI: Add per-project branch/endpoint limits.
```

not sure about this one, maybe we should call out the update of the free tier here instead.